WordPress/Classes/ViewRelated/Views/WPRichText/WPRichContentView.swift

```diff
@@ -227,10 +227,11 @@ extension WPRichContentView: WPTextAttachmentManagerDelegate {
 
         attachment.maxSize = CGSize(width: width, height: height)
 
-        if attachment.tagName == "iframe" {
-            embed.loadContentURL(URL(string: attachment.src.stringByDecodingXMLCharacters())!)
+        if let url = URL(string: attachment.src.stringByDecodingXMLCharacters()), attachment.tagName == "iframe" {
```

Comment:
Indeed it would, tho I was getting a Swift error trying to reverse the statements when I first tried.
Naturally everything works correctly now. 😬 🙄